src/test/java/seedu/address/logic/commands/EditOrderCommandTest.java

```diff
@@ -35,7 +33,7 @@
     private Model model = new ModelManager(getTypicalAddressBookOrders(), new UserPrefs());
     @Test
     public void execute_allFieldsSpecifiedUnfilteredList_success() {
-        Order editedOrder = new OrderBuilder().build(); // Creates the order in orderbuilder
+        Order editedOrder = new OrderBuilder().withDetails("1xChocCake").build(); // Creates the order in orderbuilder
```

Comment:
OrderBuilder() should take in the order to be edited so that the UUID will remain the same. The test case will fail because of differing UUID between the edited order and the original order